src/patterns/ComposedModal/_composedModal.scss

```diff
@@ -0,0 +1,50 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-composed-modal {
+  border-radius: 1rem;
+  padding: $spacing-sm 0;
+
+  &__title {
+    margin-top: $spacing-md;
+    padding: 0 $spacing-md;
+  }
+
+  &__center-align {
+    text-align: center;
+  }
+
+  &__body {
+    overflow-y: auto;
+  }
+
+  &__divider {
+    margin: 0;
+  }
+
+  &__btns-group {
+    display: flex;
+    gap: $spacing-sm;
+    justify-content: center;
+    padding: $spacing-sm $spacing-md;
+  }
+
+  &__btns-group > * {
+    flex: 1 1 50%;
+    max-width: 50%;
+  }
+
+  &__btns {
+    padding: 0 $spacing-md $spacing-sm;
+  }
+
+  &__disclaimer {
+    margin-top: $spacing-sm;
+    padding: 0 $spacing-md;
+  }
+
+  @media (max-width: $breakpoint-md) {
+    max-width: 100vw;
```

Comment:
do we need a max width if we are explicitly putting a width?

Reply:
should we possibly just have a width 100vw with the max width as to whatever the first one is from a mobile first perspecticve, then just update the max-width for the desktop breakpoint?